src/smalluniverse/orchestrators/SynchronousOrchestrator.hx

```diff
@@ -71,71 +71,73 @@ class SynchronousOrchestrator implements Orchestrator {
 	}
 
 	public function handleCommand(command:Command<Any>):Promise<Noise> {
-		switch command {
-			case Some({eventSourceClass: eventSourceClass, event: event}):
-				final subscription = getSubscriptionsToEventSource(
-					eventSourceClass
+		final handleEventPromises = command.eventsToAttempt.map(attempt -> {
```

Comment:
The diff for this file is easier to read in vscode, which can highlight the diff _other than auto-formatting changes_